eZ/Publish/Core/Persistence/Tests/TransformationProcessor/_fixtures/transformations/latin.tr.result

```diff
@@ -1,1510 +1,1984 @@
 <?php
 
-return array (
```

Comment:
This is a fixture generated automatically, please do a little research (ask around) to determine which tool generates it, because I don't remember :(

Reply:
Might be env flag picked up by the tests when running.

Reply:
I generated it using the parser so it should change a lot of lines.